### ISSUE TITLE
cyclopia in organisms without orbits or noses

### ISSUE THREAD

**Issue by srobb1** [obophenotype/upheno#304]
What is the best route to take with a cyclopic phenotype in an organism that does not have eye orbits or noses? I am working with a flatworm that has photoreceptors. The worm can have a single photoreceptor centered on the midline that we call cyclopia. Should I manually create a term or create some type of pattern to generate this phenotype term?

Here are the cyclopia terms that I have found:

Cyclopia  NCIT:C124522
http://purl.obolibrary.org/obo/NCIT_C124522
A rare congenital abnormality characterized by the failure of the embryonic prosencephalon to separate the eye orbit into two distinct cavities. Facial features tend to be absent although a proboscis has been seen to develop in conjunction.
Ontology: NCI Thesaurus OBO Edition NCIT

Cyclopia  HP:0009914
http://purl.obolibrary.org/obo/HP_0009914
Cyclopia is a congenital abnormality in which there is only one eye. That eye is centrally placed in the area normally occupied by the root of the nose.
Ontology: human phenotype ontology HP

cyclopia  MP:0005163
http://purl.obolibrary.org/obo/MP_0005163
a congenital defect characterized by the failure of the embryonic prosencephalon to properly divide the orbits of the eye into two cavities; cyclopia is a rare form of holoprosencephaly that typically presents with a median single eye or a partially divided eye in a single orbit, absent nose, and a proboscis above the eye…
Ontology: The Mammalian Phenotype Ontology MP

Thank you,
Sofia

**Comment by srobb1** [comment]
This this the right way to think about this and organize?

I have worked on the parents terms for my future cyclopia with this pattern (see bottom of this comment) using 

**has number of**  PATO:0001555
http://purl.obolibrary.org/obo/PATO_0001555
The number of parts of a particular type that the bearer entity has. This is a relational quality, and thus holds between two entities: the bearer of the quality, and the type of parts.

I currently have 

- abnormal number of photorecptors  
    -  altered number of photoreceptors
        - lacks parts or has fewer parts of type photoreceptor  
            - has fewer parts of type photoreceptor  
                - cyclopia could fit here  
             - lacks all parts of type photoreceptor ( our flatworms also have this phenotype )

```
'has_part' some ('has number of' and ('inheres_in' some 'photoreceptor') and ('qualifier' some 'abnormal'))

'has_part' some ('altered number of' and ('inheres_in' some 'photoreceptor') and ('qualifier' some 'abnormal'))

'has_part' some ('lacks parts or has fewer parts of type' and ('inheres_in' some 'photoreceptor') and ('qualifier' some 'abnormal'))

'has_part' some ('has fewer parts of type' and ('inheres_in' some 'photoreceptor') and ('qualifier' some 'abnormal'))

'has_part' some ('lacks all parts of type' and ('inheres_in' some 'photoreceptor') and ('qualifier' some 'abnormal'))
```

Proposed Pattern and classes for above lead ups to cyclopia:
```
classes:
  quantity: PATO:0001555
  abnormal: PATO:0000460
  entity: BFO:0000001

equivalentTo:
  text: "'has_part' some (%s and ('inheres_in' some %s) and ('qualifier' some 'abnormal'))"
  vars:
    - quantity
    - entity
```

**Comment by dosumis** [comment]
Cyclopia to me suggests location at the midline, not just the loss of one eye.  

We could generalize the text definition in HP to remove reference to the nose.

Cyclopia is a congenital abnormality in which there is only one eye. That eye is centrally placed in the area normally occupied by the root of the nose.
-> 
Cyclopia is a phenotype in which only a single eye is present in an animal with binocular vision - located on the midline in the area that, in a normal organism sits between the two eyes.

It would then apply to Cyclopia in fish (currently annotated in a rather unsatisfactory way):
https://zfin.org/action/figure/all-figure-view/ZDB-PUB-081203-33

We need to discuss how to record this formally. One possibility is to post-compose the entity (eye) to specify location at the midline.
  
One other question: Are the photoreceptors mentioned here single cells (CL:photoreceptor) or clusters of cells (AKA eye spots?)?

**Comment by srobb1** [comment]
@dosumis The photoreceptors are clusters of cells.  Does only light sensing ability qualify as vision?

**Comment by srobb1** [comment]
Hello. I have a new idea for my cyclopia. By the way, we have decided that 'eye' is better than 'photoreceptor'. 

` "'has_part' some ('has fewer parts of type' and ('inheres_in' some 'eye') and ('aligned with' some 'midline') and ('qualifier' some 'abnormal'))" ` 

But it would be nice to indicate that it is only 'one' and not just 'fewer parts of type'

**Comment by nicolevasilevsky** [comment]
maybe we need a new term in PATO to indicate "one"? a child of 'fewer parts of type'?

**Comment by srobb1** [comment]
@nicolevasilevsky I think that a 'one' or 'single' term would be super useful.

**Comment by nicolevasilevsky** [comment]
I'll create a ticket on the PATO tracker, and if anyone objects, they can let us know :)

**Comment by drseb** [comment]
„One“ isn’t necessarily „fewer“?!

**Comment by nicolevasilevsky** [comment]
@drseb good point. I just created a ticket in PATO, where the definition would mean that one is fewer. Based on your point though, not sure if this would be a good term?

**Comment by srobb1** [comment]
Maybe it should be something along the lines of 'abnormally one'

**Comment by srobb1** [comment]
or something like "fewer, specifically only one".

**Comment by ybradford** [comment]
for zebrafish annotations we've used 'eye' fused_with 'eye' to represent cyclopia phenotypes.
